docs/03-astropy_dev_env.rst

```diff
@@ -0,0 +1,178 @@
+How to set up a development version of Astropy
+==============================================
+
+Found a bug? Know how to fix it? See a modification you need for your
+own project? This tutorial will teach you how to set up a conda
+environment for installing and modifying a developer version of Astropy.
+
+Pre-requisites
+--------------
+
+-  `Git <https://git-scm.com/>`__ version control system. On a Mac, this
+   will come with installing the latest version of the `Xcode
+   developer <https://developer.apple.com/xcode/>`__ tools.
+-  An account on `GitHub <https://github.com/>`__
+
+This tutorial will show you how to fork, clone, and install the
+development version of Astropy from the command line (terminal).
+
+Step 1: Fork the Astropy repo on Github
+---------------------------------------
+
+-  Log into your account on Github
+-  Go to https://github.com/astropy/astropy and click on **Fork** in the
+   upper right-hand corner of the page. This will create a separate repo
+   in your Github account under github.com//astropy
+
+Step 2: Clone your forked Astropy repo to your computer
+-------------------------------------------------------
+
+-  Go to your forked version of the repository and click on **Clone or
+   download** (the green button on the upper right-hand side of the
+   page).
+-  Highlight and copy the revealed URL. If you prefer using SSH to
+   connect to Github, you can toggle between "Use SSH" and "Use HTTPS"
+   by clicking on the highlighted text in the upper right-hand corner of
+   the box.
+-  Open a terminal session
+-  Make an astropy folder in the location you would like to develop the
+   code. For example:
+
+   ::
+
+       mkdir astropy-dev
+       cd astropy-dev
+       git clone git@github.com:<user-name>/astropy.git .
+
```

Comment:
The first in a series of comments about re-naming remotes.  I'd say you want to add something like this:
```suggestion
       git remote rename origin <user-name>

Note that by default git will use the name "origin" for the your fork.  But this is often confusing because the word "origin" is ambiguous as soon as you start thinking about other forks (see below).  So renaming it to your user name will help you remember it in the future.

```